include/ccf/pal/snp_ioctl.h

```diff
@@ -29,11 +29,12 @@ namespace ccf::pal::snp
     }
   }
 
-  static std::unique_ptr<ioctl6::DerivedKey> make_derived_key()
+  static std::unique_ptr<ioctl6::DerivedKey> make_derived_key(
+    TcbVersion version = {})
```

Comment:
This looks like a separate change to the PR description - can we merge it separately?